show-karma/dao-delegates-app
# feat: gelato sdk | add delegate to registry

### Changelog

 - [X] Gelato SDK
 - [X] Delegate Registry contract
 - [X] Register delegate
 - [x] Design changes
 - [x] Show most recent statments

### To test
1 - Go to Op page
2 - Login
3 - Click edit profile
4 - Edit what you need
5 - Click save, should load a new screen to choose
6 - Select one of them and click save
7 - Select the other one and try to save.

> Note: On chain data is not being shown yet, subgraph is not working so it won't update the UI with the new data but the local save should work as usual
